docs/overview/advanced-examples.rst

```diff
@@ -170,8 +170,8 @@ First we are able to create the manager as we normally would.
     // You can allow Kompute to create the GPU resources, or pass your existing ones
     kp::Manager mgr; // Selects device 0 unless explicitly requested
 
-    // Creates tensor an initializes GPU memory (below we show more granularity)
-    auto tensor = mgr.tensor(10, 0.0);
+    // Creates tensor and initializes GPU memory (below we show more granularity)
+    auto tensor = mgr.tensorT<float>(10);
```

Comment:
I think this matches the original intent from the history of the file and the below print of `B: { 100000000, ... }` which seems to signify more than a couple elements. For such a simple example this really could be any float vector and it seems like one other interpretation for a different example was to use `mgr.tensor({ 10, 0.0 });`